cmake/scripts/run_test.sh

```diff
@@ -69,10 +69,18 @@ case $STAGE in
     rm -f stdout
 
     "$@" > stdout 2>&1
+
     RETURN_VALUE=$?
 
     [ -f output ] || mv stdout output
 
+    # Suppress meaningless output for performance tests.
+    if [ "${TEST_EXPECT}" = "PERFORMANCE" -a $RETURN_VALUE -ne 0 ]; then
+      echo "${TEST_FULL}: BUILD successful."
+      echo "${TEST_FULL}: RUN failed. ------ Performance tests disabled."
+      exit 1
```

Comment:
I don't think I understand the error message. Performance tests appear to be enabled here, the test just returned something non-zero. Did you mean this error message?